---
title: C28163
ms.date: 11/04/2016
ms.topic: reference
f1_keywords:
- C28163
helpviewer_keywords:
- C28163
ms.assetid: 24fecbde-1c96-4a45-82f7-9f47cfc0ef11
author: mikeblome
ms.author: mblome
manager: markl
ms.workload:
- multiple
ms.openlocfilehash: 3e9bd974b38647cd330c55df0b8d0063c77bff52
ms.sourcegitcommit: 485ffaedb1ade71490f11cf05962add1718945cc
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 10/16/2019
ms.locfileid: "72434561"
---
# <a name="c28163"></a>C28163
Warnung C28163: Die Funktion sollte nie innerhalb eines try/except-Blocks aufgerufen werden.

 Diese Warnung wird ausgegeben, wenn eine Funktion einen Typ hat, der niemals in einen `try/except` Block eingeschlossen werden darf, der in einem `try/except` Block gefunden wird. Das Code Analysetool hat mindestens einen Pfad gefunden, in dem die Funktion mit dem Namen in einem `try/except`-Block war.
